app/src/test/java/com/ucmobiledevelopment/freeways/IncidentTests.kt

```diff
@@ -53,7 +38,7 @@ class IncidentTests {
         assertTrue(allIncidents!!.isNotEmpty())
        var hasCaseId400434 = false
        allIncidents!!.forEach {
-             if(it.caseId.equals("400434")){
+             if(it.caseId == "400434"){
```

Comment:
We are comparing the data for structural equality. https://agrawalsuneet.github.io/blogs/equality-in-kotlin/?msclkid=208857e5af1011ec858172c495599101